sdk/eventhub/event-hubs/test/eventHubConsumerClient.spec.ts

```diff
@@ -596,5 +596,97 @@ describe("EventHubConsumerClient", () => {
       }
       logTester.assert();
     });
+
+    it("Stops receiving events if close is immediately called, single partition. #RunnableInBrowser", async function(): Promise<
+      void
+    > {
+      const partitionId = "0";
```

Comment:
Gone is the farce of getting all the partitions just to hit partitions[0]. ;)

Reply:
I prefer to think this will also catch a service-side breaking change if partition 0 ever ceases to exist ;-)